sdk/communication/test-resources.json

```diff
@@ -13,12 +13,34 @@
             "defaultValue": "communication",
             "type": "string"
         },
+        "communicationServicesEndpointSuffix": {
+            "defaultValue": ".communication.azure.com",
+            "type": "string"
+        },
         "testApplicationOid": {
             "type": "string",
             "metadata": {
                 "description": "The client OID to grant access to test resources."
             }
+        },
+        "tenantId": {
+            "type": "String",
+            "metadata": {
+                "description": "The tenant id to which the application and resources belong."
         }
+      },
+      "testApplicationId": {
```

Comment:
You have invalid json nesting here. These keys are not located within the `parameters` object.